### PR TITLE
Add an example grader config query script

### PR DESCRIPTION
**Pull request by waTeim**
Interacts with the student config database

### CODE REVIEW

File: scripts/grader_config_example.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            user=user, 
+            password=password, 
+            host=host, 
+            port=port
+        )
+
+        cur = conn.cursor()
+
+        # Execute the SQL query
+        cur.execute("SELECT * FROM student ORDER BY last_name, first_name")
```
Comment (Hoid) [inline scripts/grader_config_example.py]: can you guarantee the order in which the columns will be returned with `SELECT *`? I don't think so, I think you have to say `SELECT id, first_name, last_name, professor_onyen FROM ...`

Comment (waTeim) [reply scripts/grader_config_example.py]: I'm pretty sure that * will get the same order as that which was used in the table creation.  However more explicit is better anyway.

File: scripts/grader_config_example.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            user=user, 
+            password=password, 
+            host=host, 
+            port=port
+        )
+
+        cur = conn.cursor()
+
+        # Execute the SQL query
+        cur.execute("SELECT id,irst_name,last_name,professor_onyen FROM student ORDER BY last_name,first_name")
```
Comment (Hoid) [inline scripts/grader_config_example.py]: misspelled `first_name` :)

Comment (waTeim) [reply scripts/grader_config_example.py]: fixed

**Review by Hoid — APPROVED** [review]

**Pull request merged**